hack/demo.sh

```diff
@@ -48,10 +48,11 @@ spec:
   configuration:
     exclude:
     - not_useful
-    - test:conftest-clair
   sources:
-    - name: Default EC policy
-      policy:
+  - data:
+    - github.com/hacbs-contract/ec-policies.git//data
```

Comment:
It's pretty new, but you could consider using the bundle for this too, i.e.:
```suggestion
    - quay.io/hacbs-contract/ec-policy-data:latest
```